Tests/Test/WebTestCaseTest.php

```diff
@@ -324,9 +323,6 @@ public function testLoadFixturesFilesPaths()
         );
     }
 
-    /**
-     * @QueryCount(100)
```

Comment:
@lsmith77 I don't know anything about services and annotations reader, if nobody can fix this annotation problem, I suggest to remove this annotation feature. Did it ever worked? Has it been broken after a Symfony update?

There's still a way to check the number of queries for a specific request: https://github.com/liip/LiipFunctionalTestBundle/blob/72c8feff5fe6765e33b58397d5cbc98923d3b4b9/Tests/Test/WebTestCaseTest.php#L343